temba/orgs/tasks.py

```diff
@@ -45,3 +48,18 @@ def calculate_credit_caches():  # pragma: needs cover
 @nonoverlapping_task(track_started=True, name="squash_topupcredits", lock_key='squash_topupcredits')
 def squash_topupcredits():
     TopUpCredits.squash()
+
+
+@task(track_started=True, name='send_chatbase_logs')
+def send_chatbase_logs():  # pragma: needs cover
+    """
+    Send messages logs in batch to Chatbase
+    """
+    from temba.orgs.models import CHATBASE_API_KEY
+
+    for org in Org.objects.filter(config__icontains=CHATBASE_API_KEY):
+        cache_keys = cache.keys('org:%d:cache:chatbase_log:*' % org.id)
```

Comment:
Redis KEYS command should generally not be used in production because of performance issues (https://redis.io/commands/keys)

Better that chatbase logs are stored in a single Redis list which acts as a queue.